domains/servers.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Rage65",
+        "email": "123thetechguy@gmail.com"
+    },
+    "record": {
+        "CNAME": "rage65.duckdns.org"
```

Comment:
site seems to not be accessible

Reply:
Sorry that is my tailscale adress for my server